src/ShortScheduleCommand.php

```diff
@@ -50,9 +51,29 @@ public function isRunning(): bool
     }
 
     public function run(): void
+    {
+        $this->pendingShortScheduleCommand->getOnOneServer() ? $this->processOnOneServer() : $this->processCommand() ;
+    }
+
+    private function processOnOneServer()
+    {
+        if (Cache::missing($this->pendingShortScheduleCommand->cacheName())) {
+            Cache::add($this->pendingShortScheduleCommand->cacheName(), true, 60);
```

Comment:
Why `60`? Does the Laravel framework code also uses this value?

Reply:
The framework was using 3600 but I thought 60 seconds as this whole package is about sub minute tasks. I was thinking it should use 
`$this->pendingShortScheduleCommand->frequencyInSeconds()`

so the expiration lasts only as long as the time between checks. However this should also depend on if you care about overlaps or not. 

If it's too short could be an issue for some and same if it's too long. I assume the frameworks assumption no task will run for more than an hour.

Reply:
Ok, let's leave it at 60.

Should we also have to clean up this file when starting the loop? Right now, if we were to abort the loop at the wrong time, the file would stay, keeping to lock longer than it needs to be there.

Reply:
You mean in the situation the short-schedule:run is killed or dies then when it starts again that the command will be stuck because if the cache key is set? I have not experienced this. I am not 100% sure the running command actually terminates at the same time as as the loop.

So short-schedule:run can stop and I think the process that was running can still run and finish. It just will not be called again.

Worst case the key will expire in 1 minute. So in theory it should start running on one sever after 60 seconds in worst case scenario.